src/tools/cephfs/top/cephfs-top

```diff
@@ -253,8 +275,8 @@ class FSTop(object):
             return False
         client_metadata = stats_json[CLIENT_METADATA_KEY]
         num_clients = len(client_metadata)
-        num_mounts = len([client for client, metadata in client_metadata.items() if not
-                          metadata[CLIENT_METADATA_MOUNT_POINT_KEY] == 'N/A'])
+        num_mounts = len([client for client, metadata in client_metadata.items() if
+                          metadata.get(CLIENT_METADATA_MOUNT_POINT_KEY, 'N/A') != 'N/A'])
```

Comment:
missing `mount_point` keys would get counted as non-mount. Better to count these as unknown at display them?

Reply:
I didn't fully understand this comment. I've done a fix from what I understood. Please check and let me know, if that's what you're seeking.